client/components/shipping/services/entry.js

```diff
@@ -10,35 +12,44 @@ const ShippingServiceEntry = ( {
 	adjustment_type,
 	currencySymbol,
 	updateValue,
-} ) => (
-	<div className="wcc-shipping-service-entry">
-		<label className="wcc-shipping-service-entry-title">
-			<FormCheckbox
-				checked={ enabled }
-				onChange={ ( event ) => updateValue( 'enabled', event.target.checked ) }
+	hasError,
+} ) => {
+	const classes = classNames( 'wcc-shipping-service-entry', {
+		'wcc-error': hasError,
+	} );
+
+	return (
+		<div className={ classes }>
```

Comment:
Could simplify the code a bit by not defining classes, but just using classNames here. Optional